src/cordova/android/get_native_android_message_directory.mjs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const getNativeAndroidMessageDirectory = filepath => {
+  let [polymerLanguageFilename] = filepath
+    .split('/')
+    .at(-1)
+    .split('.');
+
+  switch (polymerLanguageFilename) {
+    case 'es-419':
+      return 'src/cordova/plugin/android/resources/strings/values-es';
+    case 'sr-Latn':
+      return 'src/cordova/plugin/android/resources/strings/values-b+sr+Latn';
```

Comment:
I can't tell if this is correct because I don't know the details of the Android resources. Where's this name coming from? Why do you have `-b-` and `-r`? Maybe a reference would be helpful.